### PR TITLE
Add tag in packit.yaml to enable cost metrics collection

### PR DESCRIPTION
**Pull request by fernflower**
Now all tft tests run by packit should be marked accordingly with a sst-upgrades tag.

OAMG-8892

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -157,12 +167,17 @@ jobs:
...
-  tf_post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys; echo 42; yum-config-manager --enable rhel-7-server-rhui-optional-rpms"
   tf_extra_params:
     environments:
       - tmt:
           context:
             distro: "rhel-7.9"
+        # tag resources as sst_upgrades to enable cost metrics collection
+        settings:
+          provisioning:
+            post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys; echo 42; yum-config-manager --enable rhel-7-server-rhui-optional-rpms"
```
Comment (pirat89) [inline .packit.yaml:178]: @fernflower  why the `echo 42`? :) is it forgotten debug cmd or it has actually some additional value?

**Review by pirat89 — APPROVED** [review]
seems good to me. untested.

File: .packit.yaml (hunk 1 above)
Comment (fernflower) [reply .packit.yaml:178]: We can safely remove that now I guess :) I will fix it in the next patch when I will be squashing 2 last commits anyway, thanks

**Comment by fernflower** [comment]
@pirat89 squashed the commits, please re-approve. 2 tests are not doing well, I am following up with the packit team on this as they started failing after move to packit and it looks like it is not because we messed up with the regex or something.
But anyway it is not due to this change.

**Pull request merged**